### PR TITLE
Organization Status Type CRUD

### PR DESCRIPTION
**Pull request by kunalsin**
-> Created views/controller for organization status type to mimic tasks page
-> Added Action Buttons for edit and delete
-> Created a new column to specify if status type is displayable

### CODE REVIEW

File: app/controllers/organization_status_types_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class OrganizationStatusTypesController < ApplicationController
+  load_and_authorize_resource
+
+  # GET /tool_types
+  # GET /tool_types.json
```
Comment (purelogiq) [inline app/controllers/organization_status_types_controller.rb]: Change these comments to be `GET /organization_status_types` and so on.

File: app/views/organization_status_types/_form.html.erb [hunk 2]
```diff
@@ -2,8 +2,9 @@
   <%= hidden_field(nil, :from_organiation_status, :value => params[:from_organiation_status]) %>
   <div class="form-inputs">
     <%= f.input :name %>
-    <%= f.input :display, :as => :boolean %>
   </div>
+  <%= f.input :display, :as => :boolean %>
+
   <div class="form-actions">
```
Comment (pkoenig10) [inline app/views/organization_status_types/_form.html.erb]: Why was this moved?

Comment (kunalsin) [inline app/views/organization_status_types/_form.html.erb]: Not sure, I changed it back and didn't do anything different so I put it back inside the div to keep it consistent
